sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AadAuthenticationProperties.java

```diff
@@ -612,24 +621,20 @@ private void validateAuthorizationClientProperties(String registrationId,
                     break;
                 case WEB_APPLICATION_AND_RESOURCE_SERVER:
                 default:
-                    LOGGER.debug("'spring.cloud.azure.active-directory.authorization-clients." + registrationId
-                        + ".authorization-grant-type' is valid.");
+                    LOGGER.debug(String.format(MATCHING_OAUTH_GRANT_TYPE_FROMAT, registrationId));
```

Comment:
why debug log can not using `LOGGER.debug("'spring.cloud.azure.active-directory.authorization-clients.{}.authorization-grant-type' is valid.", registrationId)`?

Reply:
Yes, this is not necessary.